docs/REQUIREMENTS.md

```diff
@@ -7,8 +7,197 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 ## Definitions
 
-- _**Term**: Definition._
+- **Image Compatibility specification**: A specification that describes the container image requirements for the host operating system.
 
-## User Stories
+## Member-submitted user stories (Distilled and organized by roles)
```

Comment:
```suggestion
## User stories 

> Member submitted, distilled and organized by roles
```

Reply:
It will overlap with https://github.com/opencontainers/wg-image-compatibility/pull/10/files#diff-ec360347eb95e17859836037adabfd8080b2e9336f369a6a0d4437c2d24f6a89R33

Reply:
I'd drop 12-19, and shift the remaining headings up one level, so `### Roles` becomes `## Roles`.